src/tablecloth/column/api/column.clj

```diff
@@ -0,0 +1,40 @@
+(ns tablecloth.column.api.column
+  (:require [tech.v3.dataset.column :as col]
+            [tech.v3.datatype :refer [emap]]))
+
+(defn column
+  "Create a `column` from a vector or sequence. "
+  ([]
+   (col/new-column nil []))
+  ([data]
+   (column data {:name nil}))
+  ([data {:keys [name]
+          :as options}]
+   (col/new-column name data)))
+
+;; Alias for tech.v3.dasetset.column.is-column?
+(defn column? 
+  "Return true or false `item` is a column."
+  [item]
+  (col/is-column? item))
+
+;; Alias for tech.v3.datatype.elemwise-datatype`
+(defn typeof
+  "Returns the datatype fo the elements within the column `col`."
+  [col]
+  (dtype/elemwise-datatype col))
+
+(defn typeof?
+  "True|false the column's elements are of type `dtype`"
+  [col dtype]
+  (= (dtype/elemwise-datatype col) dtype))
+
+(defn zeros
+  "Create a new column filled wth `n-zeros`."
+  [n-zeros]
+  (column (emap (constantly 0) :int64 (range n-zeros))))
```

Comment:
For `zeros` and `ones` I would use a [constant reader](https://github.com/cnuernber/dtype-next/blob/master/src/tech/v3/datatype/const_reader.clj#L18)

Reply:
Oh yes that makes sense!